src/emergency_stop_handling.c

```diff
@@ -0,0 +1,70 @@
+#include<stdio.h>
+#include<stdbool.h>
+#include"structures.h"
+#include"globals.h"
+#include"emergency_stop_handling.h"
+/**
+ * @file emergency_stop_handling.c
+ *
+ * @brief On account of emergency stop condition,All the elevators re moved to nearest floor.
+ * Passengers are removed from the elevators.
+ *
+ * @author Divyalakshmi Padmanaban  - dpadm049@uottawa.ca	
+ *
+ */
+void emergency_stop_handling(struct elevator ele[],int timer)
```

Comment:
`void emergency_stop_handling(struct elevator ele[],int *timer)` To allow for passing by reference